app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    if command.split()[0] == "mv":
```

Comment:
You need to validate command
In this line I already get an error (command = "")
Check your code for this cases:
mv file1.txt
mv file1.txt file2.txt file3.txt